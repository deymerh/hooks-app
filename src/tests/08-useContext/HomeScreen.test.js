const { mount } = require('enzyme');
const { HomeScreen } = require('../../component/08-useContext/HomeScreen');
const { UserContext } = require('../../component/08-useContext/UserContext');

describe('Pruebas en el componente <HomeScreen />', () => {

  const user = {
    name: 'Deymer',
    email: 'deymer@gmail.com'
  }
  const wrapper = mount(
    <UserContext.Provider value={{
      user
    }}>
      <HomeScreen />
    </UserContext.Provider>
  );

  test('Debe mostarse correctamente', () => {
    expect(wrapper).toMatchSnapshot();
  });

});
