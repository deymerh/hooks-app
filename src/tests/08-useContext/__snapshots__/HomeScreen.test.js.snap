// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el componente <HomeScreen /> Debe mostarse correctamente 1`] = `
<div>
  <h1>
    HomeScreen
  </h1>
  <hr />
  <pre>
    {
  "name": "Deymer",
  "email": "deymer@gmail.com"
}
  </pre>
  <div
    className="container-fluid"
  >
    <button
      className="btn btn-outline-danger"
      onClick={[Function]}
    >
      Logout
    </button>
  </div>
</div>
`;
